build/rpms/kubeadm.spec

```diff
@@ -15,12 +15,12 @@ Command-line utility for deploying a Kubernetes cluster.
 
 %install
 install -m 755 -d %{buildroot}%{_bindir}
-install -m 755 -d %{buildroot}%{_sysconfdir}/systemd/system/
-install -m 755 -d %{buildroot}%{_sysconfdir}/systemd/system/kubelet.service.d/
-install -m 755 -d %{buildroot}%{_sysconfdir}/sysconfig/
+install -m 644 -d %{buildroot}%{_sysconfdir}/systemd/system/
```

Comment:
directories should be 755, right?

Reply:
Indeed, good point :blush: 

fixed! thanks @ixdy